apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx

```diff
@@ -0,0 +1,178 @@
+                        </Diff>
+                    </Typography.Text>
+                ))}
+            </Col>
+        </Row>
+    )
+}
+
+const useChangedFieldMessagesOf = (ticketChange) => {
+    const intl = useIntl()
+    const ClientPhoneMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.clientPhone' })
+    const DetailsMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.details' })
+    const ClientNameMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.clientName' })
+    const IsPaidMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.isPaid' })
+    const IsEmergencyMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.isEmergency' })
+    const StatusDisplayNameMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.statusDisplayName' })
+    const UnitNameMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.unitName' })
+    const UnitNameChangedMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.unitName.change' })
+    const AssigneeMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.assignee' })
+    const ClassifierMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.classifier' })
+    const fields = [
+        ['clientPhone', ClientPhoneMessage],
+        ['details', DetailsMessage],
+        ['clientName', ClientNameMessage],
+        ['isPaid', IsPaidMessage],
+        ['isEmergency', IsEmergencyMessage],
+        ['statusDisplayName', StatusDisplayNameMessage],
+        ['unitName', UnitNameMessage],
+        ['assigneeDisplayName', AssigneeMessage],
+        ['classifierDisplayName', ClassifierMessage],
+    ]
+
+    const BooleanToString = {
+        isPaid: {
+            'true': intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.isPaid.true' }),
+            'false': intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.isPaid.false' }),
+        },
+        isEmergency: {
+            'true': intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.isEmergency.true' }),
+            'false': intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.isEmergency.false' }),
+        },
+    }
+
+    const formatField = (field, value) => {
+        const formatterFor = {
+            clientPhone: (field, value) => (
+                <PhoneLink value={value}/>
+            ),
+            details: (field, value) => (
+                value.length > 30 ? value.slice(0, 30) + '…' : value
+            ),
+            unitName: (field, value) => (
+                // Formally, unit name (e.g. apartment) was changed, but semantically,
+                // was changed a whole address of the ticket, so, to preserve context,
+                // the change of the unit is displayed as the change of the address
+                UnitNameChangedMessage
+                    .replace('{address}', ticketChange.ticket.property.address)
```

Comment:
use FormattedMessage component which u can import from 'react-intl'

Reply:
Used in `5ad04471`